Autopilot/Test/Src/AttitudeManager/Test_SensorFusion.cpp

```diff
@@ -1,15 +1,19 @@
 /*
-*
 * A skeleton for testing the sensor fusion algorithm
-*
+* Author: Anthony Berbari, Dhruv Rawat
```

Comment:
This was actually written by Lucy, she never stuck her name in though. You should replace my name with hers.

Reply:
Sure thing!